Detectors/FIT/FT0/simulation/src/Digitizer.cxx

```diff
@@ -28,6 +28,9 @@ ClassImp(Digitizer);
 
 Digitizer::CFDOutput Digitizer::get_time(const std::vector<float>& times, float deadTime)
 {
+  if (mEventID == 35) {
+    LOG(INFO) << " event 35 " << times.size() << " dead " << deadTime;
+  }
```

Comment:
Looks like debugging changes. Do you want to commit this?

Reply:
I removed all additional debug printout